src/components/related-grid/related-grid.scss

```diff
@@ -57,6 +57,7 @@
   .arrow {
     position: absolute;
     top: $margin-y;
+    z-index: 1;
```

Comment:
why is needed?

Reply:
the changes in visibility made the arrow be "under" the next/previous page, even though they're not visible